docs/releases/slashing-release/interfaces/core/slashing-parameter-configuration.md

```diff
@@ -0,0 +1,222 @@
+
+    /// VIEW
+
+    /**
+     * @param operator the operator to get the slashable bips for
+     * @param operatorSet the operatorSet to get the slashable bips for
+     * @param strategy the strategy to get the slashable bips for
+     * @param timestamp the timestamp to get the slashable bips for for
+     *
+     * @return slashableBips the slashable bips of the given strategy owned by
+     * the given OperatorSet for the given operator and timestamp
+     */
+    function getSlashableBips(
+        address operator,
+        OperatorSet calldata operatorSet,
+        IStrategy strategy,
+        uint32 timestamp
+    ) external returns (uint16 slashableBips);
+
+    /**
+     * @notice Get the allocatable magnitude for an operator and strategy based on number of pending deallocations
+     * that could be completed at the same time. This is the sum of freeMagnitude and the sum of all pending completable deallocations.
+     * @param operator the operator to get the allocatable magnitude for
+     * @param strategy the strategy to get the allocatable magnitude for
+     * @param numToComplete the number of pending free magnitudes deallocations to complete, 0 to complete all (uint8 max 256)
+     */
+    function getAllocatableMagnitude(
+        address operator,
+        IStrategy strategy,
+        uint8 numToComplete
+    ) external view returns (uint64);
+
+    /**
+     * @notice Get the allocation delay (in seconds) for an operator. Can only be configured one-time
+     * from calling initializeAllocationDelay.
+     * @param operator the operator to get the allocation delay for
+     */
+    function getAllocationDelay(address operator) external view returns (uint32);
+
+    /**
+     * @notice operator is slashable by operatorSet if currently registered OR last deregistered within 21 days
+     * @param operator the operator to check slashability for
+     * @param operatorSet the operatorSet to check slashability for
+     * @return bool if the operator is slashable by the operatorSet
+     */
+    function isOperatorSlashable(address operator, OperatorSet memory operatorSet) external view returns (bool);
+
+}
+```
+
+### modifyAllocations
+
+Operators call this to set the proportions of slashable stake allocated to a list of operatorSets for a set of strategies. Depending on what the new magnitude/proportion and current magnitude value is configured to, this will either create a pending allocation or deallocation. Allocations by default have a delay of 21 days but can be one-time configured by an operator. Deallocation delays all have a built-in delay of 17.5 days until they can be "completed". There is a "completing" second step to deallocations that must take place to account for the fact that deallocations are still susceptible to slashing by the operatorSet up until the completeableTimestamp for the deallocation has passed. Pending allocations on the other hand are not slashable (referring to the added increase in magnitude).
```

Comment:
there should be no default allocation delay. configutation is blocking allcoation modification.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/709/commits/25d410df3f01744b5a1ca30dda06d7750a5e5751

Reply:
we are going with configurable delays but I specified here that a initial delay is required to call modifyAllocations